pkg/kubernetes/client.go

```diff
@@ -49,3 +75,31 @@ func restConfig(kubeconfig string, url string, token string) (*rest.Config, erro
 	}
 	return &rest.Config{Host: url, BearerToken: token}, nil
 }
+
+func FetchNamespaceWithOrganizationMap(ctx context.Context, k8sClient client.Client) (map[string]string, error) {
```

Comment:
I'd probably move this to the `controlAPI` package, it doesn't really fit here.

Reply:
Doesn't make much sense in the controlAPI package either, as it's not about Namespaces on the controlAPI

Reply:
Yeah it doesn't really fit anywhere I think... ok, let's leave it here.